gulpfile.js

```diff
@@ -41,45 +35,49 @@ gulp.task('css', function () {
         cssnano()
     ];
 
-    return gulp.src('assets/css/*.css')
```

Comment:
`return`ing from a gulp task causes the `cb` to be called, which results in errors because we want it to be called once the stream created by pump has ended